src/Frame/TableFrame.py

```diff
@@ -0,0 +1,70 @@
+from tkinter import *
+import pandas as pd
+import sys
+from pathlib import Path
+
+
+#Might be a better way to do this - this was more or less how I was taught
+path_root = Path(__file__).parents[1]
+sys.path.append(str(path_root))
+print(path_root)
+from Utils.DataFrame_Windowed import DataFrame_Windowed
+
+
+ #TKinter table frame class
+class Table:
```

Comment:
You are going to need to subclass this as a Frame. See the other Frames as an example for how to mess with a frame.
```suggestion
class Table(tk.Frame):
```